app/notifiers/session_reminder_notifier.rb

```diff
@@ -17,12 +17,19 @@ class SessionReminderNotifier < ApplicationNotifier
   end
 
   notification_methods do
+    include ActionView::Helpers::DateHelper
+
     def title
       if params[:time_before_session].match?(/^0\s/)
         "Starting Now"
       else
         "Starting in about #{params[:time_before_session]}"
       end
     end
+
+    def delivered_at
+      time_difference = distance_of_time_in_words(Time.zone.now, created_at, scope: "date_time.distance_in_words.short")
```

Comment:
```suggestion
      time_difference = distance_of_time_in_words(Time.current, created_at, scope: "date_time.distance_in_words.short")
```

lets use `Time.current`
As we have a timezone configured, `#current` does the same as `#zone.now`

https://api.rubyonrails.org/classes/Time.html#method-c-current